webapp/tests/karma/ts/modules/tasks/tasks-content.component.spec.ts

```diff
@@ -472,6 +476,8 @@ describe('TasksContentComponent', () => {
 
       sinon.resetHistory();
       sinon.stub(GlobalActions.prototype, 'setEnketoError');
+      store.overrideSelector(Selectors.getSelectedTask, { _id: 'task-id'});
```

Comment:
I don't think this is needed anymore, right?

Reply:
That's right.